middleware.go

```diff
@@ -79,8 +79,6 @@ func parseRequestHeader(c *Client, r *Request) error {
 
 	if IsStringEmpty(hdr.Get(hdrUserAgentKey)) {
 		hdr.Set(hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
-	} else {
-		hdr.Set("X-"+hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
```

Comment:
@Robbilie Thank you for the PR. I have read your explanation and understood. 

Instead removing, please change to following:
```go
else {
    hdr.Add(hdrUserAgentKey, fmt.Sprintf(hdrUserAgentValue, Version))
}
```